tools/mummer.py

```diff
@@ -282,17 +282,36 @@ def scaffold_contigs_custom(self, refFasta, contigsFasta, outFasta,
                     # (# assembled segments)
                     continue
 
+                def frac_unambig(seqs):
+                    """Given a list of seqs of the same length, return the fraction of positions on which they all agree"""
+                    assert len(set(map(len, alt_seqs_f)))                    
```

Comment:
I think `assert` should only be used in tests--I think they're entirely ignored when python executes under certain circumstances.